dev/src/user.clj

```diff
@@ -9,6 +11,16 @@
          mb.hawk.assert-exprs/keep-me
          metabase.test-runner.assert-exprs/keep-me)
 
+(humane-are/install!)
+
+;; don't enable humane-test-output when running tests from the CLI, it breaks diffs.
+;;
+;; This uses [[env/env]] directly rather than [[metabase.config/is-test?]] so we can avoid loading [[metabase.config]]
+;; on REPL launch, the less `metabase.*` stuff we load on REPL launch the better because if something breaks on launch
+;; it's a big PITA to debug
+(when-not (= (env/env :mb-run-mode) "test")
+  ((requiring-resolve 'pjstadig.humane-test-output/activate!)))
+
```

Comment:
This stuff was in `metabase.test`, which gets loaded most of the time when using the REPL, but I keep finding myself in REPL sessions where I never even load that namespace when working on `metabase.lib` stuff, so I got tried of manually enabling humane test output and humane-are... I moved it to `user` so it's always enabled when we are using the `:dev` profile.